src/ir_Bryston.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright 2025 Guillaume Giraudon - Colquhoun Audio Laboratories
+
+
+/// @file
+/// @brief Support for Bryston Protocols.
+/// @note Currently only tested BP19/BR20 but should work for all Bryston Products.
+
+
+// Supports:
+//   Brand: Bryston
+
+#include <algorithm>
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRutils.h"
+
+
+// Constants
+const uint16_t kBrystonTicks = 315;  // Number of bits in a Bryston message.
+const uint16_t kBrystonHdrMark = 0;
+const uint16_t kBrystonHdrSpace = 0;
+const uint16_t kBrystonOneMark = 6 * kBrystonTicks;
+const uint16_t kBrystonOneSpace = 1 * kBrystonTicks;
+const uint16_t kBrystonZeroMark = 1 * kBrystonTicks;
+const uint16_t kBrystonZeroSpace = 6 * kBrystonTicks;
+const uint16_t kBrystonMinGap = 0;
+const uint16_t kBrystonFooterMark = 0;
+
+#if SEND_BRYSTON
+/// Send a Bryston formatted message.
+/// @param[in] data The message to be sent.
+/// @param[in] nbits The number of bits of message to be sent.
+/// @param[in] repeat The number of times the command is to be repeated.
+void IRsend::sendBryston(uint64_t data, uint16_t nbits, uint16_t repeat) {
+  Serial.printf("Sending Bryston: %016llX\n", data);
```

Comment:
Probably a leftover from debug during development that was forgotten to be removed?